utilities/cli/docker_utils.bash

```diff
@@ -45,14 +45,14 @@ function execute_docker_command {
   if [ -z "$services" ] && [ -z "$package" ]
   then
     print "Docker ${docker_compose_cmd}"
-    call "docker-compose stop" $noop
+    call "$DOCKER_COMPOSE_CMD stop" $noop
 
   else
     if [ ! -z "$services" ]
     then
       services_string=$(echo $services | tr ',' ' ')
       print "Docker ${docker_compose_cmd} service(s): ${services_string}"
-      call "docker-compose ${docker_compose_cmd} ${docker_compose_flag} ${services_string}" $noop
+      call "$DOCKER_COMPOSE_CMD ${docker_compose_cmd} ${docker_compose_flag} ${services_string}" $noop
```

Comment:
Maybe **DOCKER_COMPOSE_PREFIX_CMD** sounds better?

Reply:
I would call the "docker_compose_cmd" for "docker_compose_cmd_sub_command" instead , since the docker-compose / docker compose is the command and start/stop/.. are sub-commands, and after that flags follows.

Reply:
Sounds good, just to be a clear distinction between these two variables.

Reply:
Changed the variable name to sub-cmd variable name.